st2actions/tests/unit/test_paramiko_ssh.py

```diff
@@ -122,6 +122,31 @@ def test_key_material_argument_invalid_key(self):
                                 expected_msg, mock.connect)
 
     @patch('paramiko.SSHClient', Mock)
+    def test_key_with_passphrase(self):
+        path = os.path.join(get_resources_base_path(),
+                            'ssh', 'dummy_rsa_passphrase')
```

Comment:
Let's also add a negative test case when invalid passphrase is provided - I assume error should be propagated all the way up, right?